pkgs/by-name/gr/granted/package.nix

```
@@ -75,12 +72,11 @@ buildGoModule rec {
 
       # Insert below the #!/bin/sh shebang
       echo "$addToPath" | sed "/#!\/bin\/sh/r /dev/stdin" $src/scripts/assume >> $out/bin/assume
-    ''
-    + lib.optionalString withFish ''
+
       # Install fish script
       install -Dm755 $src/scripts/assume.fish $out/share/assume.fish
       substituteInPlace $out/share/assume.fish \
-        --replace /bin/fish ${fish}/bin/fish
+        --replace-fail "#!/bin/fish" "#!/usr/bin/env fish"
```

```suggestion
        --replace-fail "#!/bin/fish" "#!${coreutils}/bin/env fish"
```

This is more explicit.

---

Even NixOS has `/usr/bin/env`, we can expect it to be available on every distro.

It doesn't matter which `env` is used, we rely only use it to avoid adding `fish` to the closure.

---

NixOS assumes that there is an /usr/bin/env. Is that is missing you have bigger problems than this bash completion. It does not really matter to this change.